
time Stamp
uint32 SampleRate
float32[100] Voltage
float32[100] Current
